src/itensornetworkfunction.jl

```diff
@@ -73,6 +80,30 @@ function project(fitn::ITensorNetworkFunction, ind_to_ind_value_map)
   return fitn
 end
 
+" Naive integration of a function in all dimensions ∫₀¹f({r})d{r} "
```

Comment:
These integration functions should be in `elementary_operators.jl` ? Or even a new `.jl` called `integration.jl`

Reply:
They are sort of what `calculate_fxy` is, so felt right to have them side by side, but lets just export to integration then for clarity

Reply:
Yeah I see what you mean with `calculate_fxy` being there and this function not actually explicitly building an operator. Happy to keep it in `itensornetworkfunction.jl` if its less work. Probably `calculate_fxy` should be moved to something like `src/sample_itensorfunction.jl` in the future